WordPress/Classes/ViewRelated/Reader/ReaderDetailWebviewViewController.swift

```diff
@@ -208,6 +226,46 @@ class ReaderDetailWebviewViewController: UIViewController, ReaderDetailView {
         coordinator?.share(fromView: sender)
     }
 
+    /// Start following the scroll view to hide nav bar and toolbar
+    /// Only if VoiceOver is not active
+    ///
+    private func followScrollView() {
+        if isFollowingScrollView,
+            UIAccessibility.isVoiceOverRunning {
+            return
+        }
+
+        if let navigationController = navigationController as? ScrollingNavigationController {
+            navigationController.followScrollView(scrollView, delay: 50.0, followers: [NavigationBarFollower(view: toolbarContainerView, direction: .scrollDown)])
+            navigationController.shouldUpdateContentInset = false
+            isFollowingScrollView = true
+        }
+    }
+
+    /// Stop following the scroll view to hide nav bar and toolbar
+    ///
+    private func stopFollowingScrollView() {
+        if let navigationController = navigationController as? ScrollingNavigationController {
+            navigationController.stopFollowingScrollView(showingNavbar: true)
+            isFollowingScrollView = false
+        }
+    }
+
+    /// Show the nav bar when scrolling to top
+    ///
+    func scrollViewShouldScrollToTop(_ scrollView: UIScrollView) -> Bool {
+        if let navigationController = navigationController as? ScrollingNavigationController {
+            navigationController.showNavbar(animated: true, scrollToTop: true)
+        }
+        return true
+    }
+
+    /// Update scroll view insets to take into account if toolbar is visible or not
+    private func updateScrollInsets(toolbarVisibile: Bool) {
```

Comment:
```suggestion
    private func updateScrollInsets(toolbarVisible: Bool) {
```

Reply:
Done in 30fe93e6b3173d9ca702d534d42a65ddc86e5b9b